packages/legend-application-studio/src/components/editor/edit-panel/project-configuration-editor/ProjectConfigurationEditor.tsx

```diff
@@ -445,6 +433,237 @@ const ProjectDependencyEditor = observer(
+                  isLatestVersion,
+              },
+              {
+                'project-configuration-editor__platform__configuration__label__version--outdated':
+                  isLatestVersion === false,
+              },
+            )}
+          >
+            {platform.version}
+          </div>
+        </div>
+        {isLatestVersion === false && (
+          <button
+            className="project-configuration-editor__platform__configuration__label__version__update-btn"
+            tabIndex={-1}
+            title={`Current platform configuration is outdated. Click to update to the latest version`}
+            onClick={updateLatest}
+          >
+            Update to latest version
+          </button>
+        )}
+      </div>
+    );
+  },
+);
+
+const ProjectPlatformVersionEditor = observer(
+  (props: { projectConfig: ProjectConfiguration; isReadOnly: boolean }) => {
+    const { projectConfig, isReadOnly } = props;
+    const editorStore = useEditorStore();
+    const applicationStore = useApplicationStore();
+
+    const isNotEmptyPlatforms = (
+      platformConfigurations: PlatformConfiguration[] | undefined,
+    ): boolean => {
+      if (platformConfigurations) {
+        return platformConfigurations.every(
+          (p) => p.name !== undefined && p.version !== undefined,
+        );
+      }
+      return false;
+    };
+
+    const convertPlatformtoPlatformConfiguration = (
+      platforms: Platform[] | undefined,
+    ): PlatformConfiguration[] | undefined => {
+      if (platforms) {
+        return platforms.map(
+          (platform) =>
+            new PlatformConfiguration(platform.name, platform.platformVersion),
+        );
+      } else {
+        return undefined;
+      }
+    };
+
+    const platformConfigurations = projectConfig.platformConfigurations;
+    const defaultPlatforms = convertPlatformtoPlatformConfiguration(
+      editorStore.sdlcServerClient.platforms,
+    );
+
+    const isLatestVersion = (platform: PlatformConfiguration): boolean =>
+      defaultPlatforms?.find(
+        (defaultPlatform) => defaultPlatform.name === platform.name,
+      )?.version === platform.version;
+
+    const seeDocumentation = (): void => {
+      applicationStore.assistantService.openDocumentationEntry(
+        LEGEND_APPLICATION_DOCUMENTATION_KEY.QUESTION_WHAT_IS_PROJECT_PLATFORM_CONFIGURATIONS,
+      );
+    };
+
+    const toggleOverridePlatformConfigurations = (): void => {
+      if (!isNotEmptyPlatforms(projectConfig.platformConfigurations)) {
+        editorStore.setActionAlertInfo({
+          message:
+            "This is an advanced configuration option that means you will no longer be relying on the latest platform versions but fixing your platform dependencies' versions to the ones you are using at this point in time. ",
+          prompt: 'Do you still want to proceed?',
+          type: ActionAlertType.CAUTION,
```

Comment:
Probably we don't need this anymore. We have a pretty detailed description/instruction already, so people should be able to make informed decision at this point